WordPress/src/main/res/layout/domain_suggestions_fragment.xml

```diff
@@ -0,0 +1,124 @@
+                xmlns:app="http://schemas.android.com/apk/res-auto"
+                xmlns:tools="http://schemas.android.com/tools"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="@android:color/white">
+
+    <LinearLayout
+        android:id="@+id/search_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginTop="@dimen/margin_extra_small"
+        android:gravity="center_vertical"
+        android:orientation="horizontal"
+        android:paddingBottom="@dimen/margin_extra_large"
+        android:paddingEnd="@dimen/margin_extra_large"
+        android:paddingStart="@dimen/margin_large"
+        android:paddingTop="@dimen/margin_extra_large">
+
+        <FrameLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/margin_extra_extra_medium_large">
+
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/margin_extra_extra_small"
+                android:contentDescription="@null"
+                android:src="@drawable/ic_search_white_24dp"
+                android:tint="@color/neutral"/>
+
+            <ProgressBar
+                android:id="@+id/suggestion_progress_bar"
+                android:layout_width="@dimen/progress_spinner_small"
+                android:layout_height="@dimen/progress_spinner_small"
+                android:background="@android:color/white"
+                tools:visibility="gone"/>
+        </FrameLayout>
+
+        <EditText
+            android:id="@+id/domain_suggestion_keyword_input"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/margin_extra_small"
+            android:background="@android:color/transparent"
+            android:clickable="false"
+            android:hint="@string/suggestions_search_hint"
+            android:imeOptions="actionDone"
+            android:inputType="text"
+            android:maxLines="1"
+            android:paddingStart="@dimen/margin_extra_extra_small"
+            android:textColorHint="@color/neutral_400"
+            android:textSize="@dimen/text_sz_medium"
+            tools:ignore="Autofill,RtlSymmetry"/>
+    </LinearLayout>
+
+    <View
+        android:id="@+id/divider"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_below="@+id/search_container"
+        android:layout_marginEnd="@dimen/margin_extra_large"
+        android:layout_marginStart="@dimen/margin_extra_large"
+        android:background="@color/neutral_0"
```

Comment:
```suggestion
        android:background="@color/divider"
```

We should revert `@color/neutral_0` back to `@color/divider`.

I know [@SylvesterWilmott requested to mimic domain selection on site creation](https://github.com/wordpress-mobile/WordPress-Android/pull/9715#issuecomment-488644622), but the `divider` color was created specifically to be used for dividers like this.  Rather than change this layout to match site creation, we should update the `android:layout_height="1dp"` attribute to `android:layout_height="@dimen/list_divider_height"` and the `android:background="@color/neutral_0"` attribute to `android:background="@color/divider"` for the divider view in the `new_site_creation_search_input_item ` layout.  If we intend to use `neutral_0` for dividers, we should update the `divider` color for consistency throughout the app.  We should also consult @mattmiklic for semantic color changes since this was just released.  All of that should probably be done in a separate pull request too.

Reply:
Sounds good to me 👍